README.md

```diff
@@ -60,6 +61,29 @@ fuck go back
 
 they're the same picture
 ```
+4. Express your feelings during a math exam:
+```
+upgrade
+what can I say except A
+fuck go back
+```
```

Comment:
This example is way better than example two, so I would either exchange 4 and 2 or remove my "setting register to 0" example entirely